### PR TITLE
Added NEST.py to tests/ and updated bindings.cpp

### PR DESCRIPTION
**Pull request by infophysics**
Updated bindings.cpp to include py::nodelete in the VDetector binding.  This solved issue #11 

Added NEST.py to tests/ for conducting unittests which are related to issue #6 and #12

### CODE REVIEW

File: tests/NEST.py [hunk 1]
```diff
@@ -0,0 +1,659 @@
+from nestpy import nestpy
+from types import *
+import unittest
+from enum import Enum
+
+import sys
+
+def trace(frame, event, arg):
+    print "%s, %s:%d" % (event, frame.f_code.co_filename, frame.f_lineno)
```
Comment (tunnell) [inline tests/NEST.py]: Can you use print() with the brackets so it works in Python3?

File: tests/NEST.py [hunk 2]
```diff
@@ -0,0 +1,659 @@
...
+    # y.print_values()
+    # z = QuantaResult(0, 1, 2, 3.)
+    # z.print_values()
+    # x = NESTresult(y, z, [1, 1, 1])
+    #
+    # detec = VDetector()
+    # detec.initialization()
+    # nest = NESTcalc(detec)
+    # int_type = INTERACTION_TYPE(0)
+    # result = nest.FullCalculation(int_type, 1., 2., 3., 4, 5)
```
[the remark this answers is not in the document]
Comment (infophysics) [reply tests/NEST.py]: Removed

File: tests/NEST.py [hunk 3]
```diff
@@ -0,0 +1,659 @@
...
+
+    #   CalcElectronLET
+    def CalcElectronLET(self, E):
+        return self.calc.CalcElectronLET(E)
+
+    #   GetDetector
+    def GetDetector(self):
+        return VDetector(self.calc.GetDetector())
+
+class NESTcalcTestt(unittest.TestCase):
```
[the remark this answers is not in the document]
Comment (infophysics) [reply tests/NEST.py]: Fixed

File: tests/NEST.py [hunk 4]
```diff
@@ -0,0 +1,659 @@
+from nestpy import nestpy
+from types import *
+import unittest
+from enum import Enum
+
+import sys
+
+def trace(frame, event, arg):
```
[the remark this answers is not in the document]
Comment (infophysics) [reply tests/NEST.py]: Removed

File: tests/NEST.py (hunk 1 above)
Comment (infophysics) [reply tests/NEST.py]: Removed

File: src/nestpy/bindings.cpp [hunk 5]
```diff
@@ -53,7 +53,7 @@ PYBIND11_MODULE(nestpy, m) {
     .export_values();
 
   //	Binding for the VDetector class
-  py::class_<VDetector>(m, "VDetector")
+  py::class_<VDetector, std::unique_ptr<VDetector, py::nodelete>>(m, "VDetector")
```
[the remark this answers is not in the document]
Comment (infophysics) [reply src/nestpy/bindings.cpp]: Done

File: src/nestpy/NEST.cpp [hunk 6]
```diff
@@ -1258,11 +1258,11 @@ vector<double> NESTcalc::SetDriftVelocity_NonUniform(double rho, double zStep,
 
 vector<double> NESTcalc::xyResolution(double xPos_mm, double yPos_mm,
                                       double A_top) {
+
```
Comment (tunnell) [inline src/nestpy/NEST.cpp]: You modified NEST.cpp?  I guess just spacing?

**Pull request merged**